jstest/testrunner/devices/connections/sshcom.py

```diff
@@ -24,25 +24,36 @@ class SSHConnection(object):
     '''
     def __init__(self, device_info):
         self.username = device_info['username']
+        self.password = device_info['password']
         self.ip = device_info['ip']
         self.port = device_info['port']
         self.timeout = device_info['timeout']
+        self._no_exec_command = device_info['_no_exec_command']
```

Comment:
It would be nice if this variable could have a comment about its purpose. For example: `FIXME: the exec_command of Paramico SSH library can't be used with the emulated SSH server.` (or something else).